contrib/kafka/pkg/reconciler/reconciler.go

```diff
@@ -0,0 +1,168 @@
+/*
+Copyright 2019 Google LLC
```

Comment:
is this right? Should be knative authors?

Reply:
sorry, copy/paste error... thanks!